registry/remote/example_test.go

```diff
@@ -82,6 +117,49 @@ func TestMain(m *testing.M) {
 			w.WriteHeader(http.StatusCreated)
 		case p == fmt.Sprintf("/v2/%s/manifests/%s", exampleRepositoryName, exampleRefereceDigest) && m == "PUT":
 			w.WriteHeader(http.StatusCreated)
+		case p == fmt.Sprintf("/v2/%s/manifests/%s", exampleRepositoryName, exampleReferenceManifestDescriptor.Digest) && m == "GET":
+			w.Header().Set("Content-Type", artifactspec.MediaTypeArtifactManifest)
+			w.Header().Set("Content-Digest", string(exampleReferenceManifestDescriptor.Digest))
+			w.Header().Set("Content-Length", strconv.Itoa(len(exampleReferenceManifest)))
+			if m == "GET" {
+				w.Write(exampleReferenceManifest)
+			}
+		case p == fmt.Sprintf("/v2/%s/manifests/%s", exampleRepositoryName, exampleSignatureManifestDescriptor.Digest) && m == "GET":
+			w.Header().Set("Content-Type", artifactspec.MediaTypeArtifactManifest)
+			w.Header().Set("Content-Digest", string(exampleSignatureManifestDescriptor.Digest))
+			w.Header().Set("Content-Length", strconv.Itoa(len(exampleSignatureManifest)))
+			if m == "GET" {
+				w.Write(exampleSignatureManifest)
+			}
+		case p == fmt.Sprintf("/v2/%s/manifests/%s", exampleRepositoryName, exampleSBoMManifestDescriptor.Digest) && m == "GET":
+			w.Header().Set("Content-Type", artifactspec.MediaTypeArtifactManifest)
+			w.Header().Set("Content-Digest", string(exampleSBoMManifestDescriptor.Digest))
+			w.Header().Set("Content-Length", strconv.Itoa(len(exampleSBoMManifest)))
+			if m == "GET" {
+				w.Write(exampleSBoMManifest)
+			}
+		case p == fmt.Sprintf("/v2/%s/_oras/artifacts/referrers", exampleRepositoryName):
+			q := r.URL.Query()
+			n, err := strconv.Atoi(q.Get("n"))
+			if err != nil || n != 2 {
+				w.WriteHeader(http.StatusBadRequest)
+			}
```

Comment:
We don't need to set `n` from client side for this example. Hence we don't need to check it.

Reply:
Deleted.